### PR TITLE
Update example files and docs.

### PR DESCRIPTION
**Pull request by tomplus**
Update example json and readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,40 +27,44 @@ present, it is used to import additional properties.
 [Example file](./example/cubes.json)
 
 Entries:
-## `materials` list
+### `materials` list
```
Comment (wmatyjewicz) [inline README.md:30]: There is no `##` level, but it looks better when this is on the 3rd level, so 👍

File: README.md [hunk 2]
```diff
@@ -71,65 +75,167 @@ is loaded.
...
+a given point in 3D space, the ambient occlusion has no effect on this point.
+
+`texture` has the following properties:
+
++ `fileName`: required, name of the file inside the archive that
+stores the sky texture.
++ `yawRotation` optional number in `[0,360]` range that specifies the
+rotation of the sky texture in degrees. Defaults to `0`.
+
+### `bake` object
```
Comment (wmatyjewicz) [inline README.md]: This is for cloud API only.

File: README.md [hunk 3]
```diff
@@ -71,65 +75,167 @@ is loaded.
...
+
+`bake` is an optional object that allows to control lightmap baking
+quality. It has the following property:
+
++ `quality`: a string, one of `"draft"`, `"low"`, `"medium"`,
+`"high"`, "`super`". Each setting corresponds to different lightmap
+baking parameters (the number of samples, bounces and the maximum
+number of lightmaps).
+
+### `panoramas` list
```
Comment (wmatyjewicz) [inline README.md]: This is for cloud API only.

File: example/cubes.json [hunk 4]
```diff
@@ -119,5 +237,34 @@
         }
       ]
     }
-  ]
+  ],
+
+  "panoramas": [
```
Comment (wmatyjewicz) [inline example/cubes.json]: This is for cloud API only.

File: example/cubes.json [hunk 5]
```diff
@@ -119,5 +237,34 @@
...
+      "rotation": -90,
+      "position": [
+        -0.115,
+        0,
+        0
+      ]
+    }
+  ],
+
+  "bake": {
```
Comment (wmatyjewicz) [inline example/cubes.json]: This is for cloud API only.

**Review by wmatyjewicz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tomplus) [reply README.md]: Fixed, sections removed from doc and example.

**Review by wmatyjewicz — APPROVED** [review]

**Pull request merged**